lib/pbench/test/unit/server/query_apis/test_datasets_list.py

```diff
@@ -53,12 +44,14 @@ def test_query(
         """
         payload = {
             "user": user,
+            "access": "private",
             "controller": "dbutenho.csb",
             "start": "2020-08",
             "end": "2020-10",
         }
         if user == "no_user":
             payload.pop("user", None)
```

Comment:
In `lib/pbench/test/unit/server/query_apis/test_datasets_detail.py`, you've switched from `pop()` to `del`...did you want to do that here?

Reply:
I didn't look here, since I didn't touch it. In fact, I thought I'd *restored* a `del` in the other case which I'd temporarily removed in my travels ... I forgot the original had for some reason used a `pop`. Hmm.